assets/js/blocks/cart-checkout/payment-methods/style.scss

```diff
@@ -89,12 +89,12 @@
 		}
 		.wc-block-gateway-input.focused.empty,
 		.wc-block-gateway-input:not(.empty) {
-			& + label {
+			+ label {
```

Comment:
Same question here and in the following lines. Has the meaning of `&` changed in some way I am unaware of, perhaps?

Reply:
resolving in favour of first occurrence :)